### DISCUSSION TITLE
Add to the documentation how to implement an OSS authentication with a provider (ex. google) with a next/nuxt client etc.

### DISCUSSION THREAD

**Discussion by iddar** [directus/directus#22790] [Feature Requests]
### Summary

I have been trying to follow all the guides and comments in the communities for a few weeks and I cannot implement a login with an OSS (aka Google) from my app in Nuxt.js. The site guides are incomplete and only indicate the enpoint but not the complete path. To complete the mission, I manage to do the direct request but when I return to my site the session is not shared.

### Basic Example

Have a direct instance with Auth with Google and an application in nuxt, try to implement a flirt with Google from it.

### Motivation

I have wasted weeks wandering around and I can't find a clear solution.

### Detailed Design

Documentation

### Should have:
A clear guide on how to do this type of integration or make it clear if this is impossible.

### Should have:
Indicate that ENV_VARS, endpoint and permissions must be modified/used for this to work

### Could have:
Examples, code repository

### Drawbacks

none

### Alternatives

I want for that

### Adoption Strategy

This task can be part of the community or someone who is in charge of the auth area

**Comment by joselcvarela** [comment]
Hello @iddar 
I also had some issues trying to setup SSO with external application but mainly because Directus relies on cookies and cookies are kinda difficult to manage between different domains.
Here's some guidance that may be helpful:
https://github.com/joselcvarela/directus-examples/tree/main/sso-integration

**Reply by iddar** [reply]
Awesome great guide, thaks 

Do you know if I can authorize multiple domains, another point that I want to enable is the authentication with OpenAI / GPT, by chance, do you have any ideas?

https://platform.openai.com/docs/actions/authentication
